src/main/scala/spark/dicom/DicomSparkMapper.scala

```diff
@@ -157,7 +157,9 @@ object DicomSparkMapper {
             UTF8String.fromString(
               Option(attrs.getSequence(tag))
                 .map(seq => {
-                  val ja = DicomJson.seq2jsonarray(seq)
+                  val ja = DicomJson.deepRenameJsonKeyTagToKeyword(
+                    DicomJson.seq2jsonarray(seq)
+                  )
```

Comment:
Could we make the renaming optional ? Maybe in another PR ?

Reply:
Yes, will pick it up in the next PR